garak/command.py

```diff
@@ -117,7 +123,7 @@ def end_run():
     print(f"📜 report html summary being written to {digest_filename}")
     try:
         write_report_digest(_config.transient.report_filename, digest_filename)
-    except Exception as e:
+    except BaseException as e:
```

Comment:
Why `BaseException`? 

Catching `Exception` is very broad already, this only adds `GeneratorExit`, `KeyboardInterrupt` and `SystemExit` as far as I can tell all of which I would expect to immediately terminate vs being able to recover in any meaningful way.

Reply:
reverted.

Would like to scope this down, actually - I am not sure what is being caught here, because `Exception` is an uninformative signal